### PR TITLE
Refactor VPC importing

### PR DESCRIPTION
**Pull request by michaelbeaumont**
### Checklist
- [ ] Added tests that cover your change (if possible)
- [ ] Added/modified documentation as required (such as the `README.md`, or the `userdocs` directory)
- [ ] Manually tested
- [ ] Made sure the title of the PR is a good description that can go into the release notes
- [ ] (Core team) Added labels for change area (e.g. `area/nodegroup`) and kind (e.g. `kind/improvement`)

### BONUS POINTS checklist: complete for good vibes and maybe prizes?! :exploding_head:
- [ ] Backfilled missing tests for code in same general area :tada:
- [ ] Refactored something and made the world a better place :star2:

### CODE REVIEW

File: pkg/apis/eksctl.io/v1alpha5/vpc.go [hunk 1]
```diff
@@ -64,6 +64,48 @@ func (m *AZSubnetMapping) SetAZ(az string, spec Network) {
...
+	if m == nil {
+		return nil
+	}
+	subnets := []string{}
+	for _, s := range *m {
+		if s.CIDR != nil {
+			subnets = append(subnets, s.CIDR.String())
+		}
+	}
+	return subnets
```
Comment (aclevername) [inline pkg/apis/eksctl.io/v1alpha5/vpc.go:92]: do we need to consider if a subnet has provided its ID and the cidr range?

Comment (michaelbeaumont) [reply pkg/apis/eksctl.io/v1alpha5/vpc.go:92]: hmm I suppose not. that reminds me also I need to add validation

Comment (michaelbeaumont) [reply pkg/apis/eksctl.io/v1alpha5/vpc.go]: going to look at that in a new PR though

Comment (aclevername) [reply pkg/apis/eksctl.io/v1alpha5/vpc.go]: :+1:

**Review by aclevername — APPROVED** [review]
LGTM

**Pull request merged**